operator/src/main/java/oracle/kubernetes/operator/steps/ReadHealthStep.java

```diff
@@ -183,15 +312,114 @@ public NextAction apply(Packet packet) {
       }
     }
 
+    /**
+     * Returns the URL to access the Service; using the Service clusterIP and port. If the service
```

Comment:
I know you probably copied this method, but this text doesn't describe what the method is doing. It looks like it's figuring out the URL of where the server can receive admin traffic.

Reply:
This method was originally in HttpClient class and I just moved it to ReadHealthStep class (since HttpClient class was removed). I discussed the Java doc text with Anthony and have changed it based on that discussion. Please let me know if this is not clear enough.